src/elements/content-preview/ContentPreview.js

```diff
@@ -802,6 +802,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
             showProgress: false,
             skipServerUpdate: true,
             useHotkeys: false,
+            contentInsightsAdditionalData: {},
```

Comment:
Is this needed for the back button change?

Reply:
It was an additional change that I made to pass through some data from EUA to the content-preview

Reply:
Gotcha. Let's go with `contentInsightsOptions` or something, then.